resourcemanager/resourcemanager.go

```diff
@@ -246,3 +247,20 @@ func (rm ResourceManager) saveResourceData(filePath string, resourceData []byte)
 	}
 	return nil
 }
+
+// SaveStringToFile saves a string to file
+func (rm ResourceManager) SaveStringToFile(filePath string, data string) error {
```

Comment:
Maybe this should be moved to `util` package. You could put this in separate file or in `util/utils.go`.

Reply:
Good idea